core/templates/pages/topic-editor-page/services/topic-editor-state.service.spec.ts

```diff
@@ -200,6 +256,15 @@ describe('Topic editor state service', function() {
       'TopicRightsBackendApiService',
       [fakeTopicRightsBackendApiService][0]);
   }));
+  beforeEach(
+    angular.mock.module('oppia', TranslatorProviderForTests));
+  beforeEach(angular.mock.module('oppia', function($provide) {
+    fakeEditableStoryBackendApiService = (
```

Comment:
Let's combine these with the beforeEach above (after line 257)?

Reply:
Done.